From fcd9bafc69b924f23b9551642beeb6b3c357938e Mon Sep 17 00:00:00 2001
From: Robby Cai <robby.cai@nxp.com>
Date: Tue, 30 Aug 2016 18:06:46 +0800
Subject: [PATCH 1140/5242] MLK-13142 media: capture: correct the AVDD supply
 for camera

commit  19207d820caeabb227ae45448a9c1a0e6a435704 from
https://source.codeaurora.org/external/imx/linux-imx.git

On i.MX6QP SabreSD board, AVDD supply is changed to VGEN6.

Signed-off-by: Robby Cai <robby.cai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qp-sabresd.dts |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qp-sabresd.dts b/arch/arm/boot/dts/imx6qp-sabresd.dts
index 559d2d5..aa8bc7e 100644
--- a/arch/arm/boot/dts/imx6qp-sabresd.dts
+++ b/arch/arm/boot/dts/imx6qp-sabresd.dts
@@ -109,10 +109,12 @@
 };
 
 &ov564x {
+	AVDD-supply = <&vgen6_reg>; /* 2.8v */
 	DOVDD-supply = <&sw4_reg>; /* 1.8v */
 };
 
 &ov564x_mipi {
+	AVDD-supply = <&vgen6_reg>; /* 2.8v */
 	DOVDD-supply = <&sw4_reg>; /* 1.8v */
 };
 
-- 
1.7.9.5

